from domain.ValidatorException import ValidatorException


# class EventValidator:
#
#     def validate(self, event):
#         errors = ""
#         if event.get
    # def __init__(self):
    #     self.__errors = []
    #
    # def validate(self, event):
    #     self.__errors = []
    #     self.validate_date(event.get_date())
    #     self.validate_time(event.get_time())
    #     self.validate_description(event.get_description())
    #     if len(self.__errors) > 0:
    #         raise ValidatorException(self.__errors)
    #     else:
    #         return True